src/main/java/ShortSet.java

```diff
@@ -1,3 +1,5 @@
+import java.util.*;
+
 public class ShortSet {
   public static void main (String[] args) {
```

Comment:
Good job! The static analysis tool Error Prone reported that the below error [1] used to be here, but you removed it. Error Prone also found similar issues in https://github.com/chbrown13/RecommenderTest/blob/cd986d54720c0a6f8e2a8d6f6ad61d800ef2139a/src/main/java/ShortSet.java#L3 and https://github.com/chbrown13/RecommenderTest/blob/37a92d8c9cf1f132a59dc2dea77b99baa3ebf25e/RecommenderTest/src/main/java/ShortList.java#L3.Check out http://errorprone.info for more information.


[1] ```RecommenderTest/src/main/java/ShortSet.java:4: error: cannot find symbol
    Set<Short> s = new HashSet<>();
    ^
  symbol:   class Set
  location: class ShortSet```